act-rules-format.md

```diff
@@ -497,6 +523,69 @@ Background (optional) {#background}
+
+For each implementation the following data <em class="rfc2119">may</em> be provided. If included, consistency and coverage <em class="rfc2119">must</em> be determined as defined in this section.
+
+- Name of implementation
+- Name of the vendor
+- [Consistency](#impl-consistency)
+- [Coverage](#impl-coverage) 
+- Version used in testing consistency and coverage
+- [test modes](https://www.w3.org/TR/EARL10-Schema/#TestMode) of the set implementation procedures (e.g. automatic, manual, semiAuto)
+
+### Consistency ### {#impl-consistency}
+
+Consistency says how well a set of [=implementation procedures=] are aligned with the intent of an ACT Rule. Consistency is determined by comparing the [=outcome=] that implementation procedures return on the test cases to the expected outcomes of the test cases. In determining consistency, the `passed` and `inapplicable` outcomes are interchangeable. This supports implementations that do not distinguish between `passed` and `inapplicable`, and allows implementation procedures to be scoped different from an ACT Rule.
+
+In addition to `passed`, `failed`, and `inapplicable`, implementations can report `cantTell` and `untested` as an [=outcome=]. If no outcome is given for a test case, it is considered `untested`.
+
+There are three levels of consistency; complete, partial, and minimal. Consistency levels are defined in the following way. The consistency of a set of implementations procedures is the first level in the following list that matches the definition.
+
+1. <dfn>Complete</dfn> implementations consider all aspects of a rule's applicability and expectations. For the [=outcomes=] reported for some set of [=implementation procedures=], all the following are true:
+
+    - All `failed` test cases are reported as `failed`, or `cantTell` by at least one procedure; and
+    - all `passed` and `inapplicable` test cases are reported as `passed`, or `inapplicable`, or `cantTell` by all procedures; and
+    - `passed` and `failed` test cases are not all reported as `cantTell` by all procedures; and
+    - the implementation procedures report a [=consistent accessibility requirements mapping=].
+
+2. <dfn>Partial</dfn> implementations consider some, but not all aspects of a rule's applicability and expectations. For the [=outcomes=] reported for some set of [=implementation procedures=], all the following are true:
+
+    - All `passed` and `inapplicable` test cases are reported as `passed`, `inapplicable`, `cantTell`, or `untested`; and
```

Comment:
There's another problem with these. We're assuming each procedure reports only 1 outcome per test case. That's not necessarily the case. If a passed example is reported both as passed, and as failed, that's still a problem.